src/window.cpp

```diff
@@ -98,10 +101,10 @@ window_impl::window_impl(int pWidth, int pHeight, const char* pTitle,
         }
     }
 
-    // Set context (before glewInit())
+    /* Set context (before glewInit()) */
     MakeContextCurrent(this);
 
-    //GLEW Initialization - Must be done
+    /* GLEW Initialization - Must be done */
```

Comment:
@pavanky 
Is '//' is better or something? or this is something we follow across projects. The reason i ask is, I made it `/**/` instead of `//` is, `//` can accidentally result in commented out valid code. Say, if by mistake the programmer writes the following instead of the later
```
// update variable var = val;
```
```
// update variable
var = val;
```
I understand these kind of mistakes are very unlikely, but `/**/` would eliminate any error at all.